resources/model/OpenStudio.idd

```diff
@@ -5432,11 +5432,21 @@ OS:Space,
   A10, \field Building Unit Name
        \type object-list
        \object-list BuildingUnitNames
-  N5; \field Volume
+  N5, \field Volume
        \type real
        \units m3
        \autocalculatable
        \default autocalculate
+  N6, \field Ceiling Height
+       \type real
+       \units m
+       \autocalculatable
+       \default autocalculate
+  N7; \field Floor Area
+       \type real
+       \units m2
+       \autocalculatable
+       \default autocalculate
```

Comment:
(Side note; no action required: I know you're following the `Volume` pattern, though I think I would rather have it as \required-field and hardset to "Autocalculate" in the ctor)